website/blog/testing.md

```diff
@@ -0,0 +1,604 @@
+# Lower Test Toil With Better Local Testing
+
+[Service Weaver][tutorial] is a programming framework that makes it easy to
+write, test, and deploy distributed applications. In [previous blog
+posts][blog], we discussed how to write and deploy Service Weaver applications.
+In this blog post, we'll focus on testing. Specifically, we'll explore
+
+- how Service Weaver lets you **unit test** system components using regular Go
+  unit tests;
+- how Service Weaver makes it significantly easier to write **integration
```

Comment:
can we add some of the numbers you have for integration testing like how difficult is to do this today?

Reply:
Done.